packages/react-components/lib/title-bar.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { createStyles, makeStyles, AppBar, Theme, Toolbar, Typography } from '@material-ui/core';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Badge from '@material-ui/core/Badge';
+
+export interface TitleBarProps {
+  logoPath: string;
+  tabNames: string[];
+  children?: React.ReactNode;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    logo: {
+      maxWidth: 120,
+      marginLeft: theme.spacing(2),
+      marginRight: theme.spacing(2),
+      opacity: 1,
+    },
+    subtitle: {
+      textAlign: 'right',
+      flexGrow: 1,
+    },
+    tab: {
+      border: '0.25px solid rgba(251, 252, 255, 0.5)',
+    },
+    toolbar: {
+      flexGrow: 1,
+    },
+  }),
+);
+
+const TitleBar = (props: TitleBarProps): React.ReactElement => {
+  const { logoPath, children, tabNames } = props;
+  const classes = useStyles();
+
+  const [value, setValue] = React.useState(0);
+
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+    setValue(newValue);
+  };
+
+  const populateTabs = () => {
+    const tabs = tabNames.map((value, index) => (
+      <Tab key={index} label={value} value={index} className={classes.tab} />
+    ));
+    return tabs;
+  };
+
+  return (
+    <AppBar id="appbar" position="static">
+      <Tabs
+        variant="scrollable"
+        scrollButtons="auto"
+        value={value}
+        onChange={handleChange}
+        aria-label="navigation-tabs"
+      >
+        <Tab
+          label={
+            <Badge badgeContent={<img src={logoPath} alt="logo" className={classes.logo} />} />
+          }
+        />
```

Comment:
This makes the logo clickable, which is pretty nice. Do you have a plan on what clicking it would do?

Reply:
Previously I thought having the action to be clicking on the logo would bring it back to the 'homepage', which is the first building tab for now, but I'm not sure if it's necessary or it will be confusing. Although this is conventional in most web pages!